### PR TITLE
chore(mailmap): Update Nicolas' email address

### PR DESCRIPTION
**Pull request by nnobelis**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -29,7 +29,7 @@ Martin Nonnenmacher <martin.nonnenmacher@bosch.io> <mnonnenmacher@users.noreply.
 Mikko Murto <mikko.murto@doubleopen.org> <mikko.murto@hhpartners.fi>
 Mo Khan <mo.khan@gmail.com>
 Maximilian Huber <maximilian.huber@tngtech.com> <gh@maxhbr.de>
-Nicolas Nobelis <nicolas.nobelis@bosch.com> <nicolas.nobelis@bosch.io>
+Nicolas Nobelis <nicolas.nobelis@bosch.com> <nicolas.nobelis@bosch.com>
```
Comment (sschuberth) [inline .mailmap:32]: There should be no need to list your `.com` email address twice, or? Both Bosch addresses already get consolidated to `Nicolas Nobelis <nicolas.nobelis@bosch.com>`, so it should be good as-is.

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (nnobelis) [reply .mailmap]: Oh sorry Monday morning...

The point is the .io address should not be used anymore. Should I remove the mail or leave as is as you said ?

Comment (nnobelis) [reply .mailmap]: Is is important regarding past commits ?

Comment (sschuberth) [reply .mailmap]: > Is is important regarding past commits ?

Yes, so we should keep it.

**Comment by nnobelis** [comment]
Not needed (see comment).

**Pull request closed without merging**